config/ember.js

```diff
@@ -1,9 +1,18 @@
 module.exports = {
-  extends: require.resolve('./base.js'),
+  extends: [
+    'plugin:ember/recommended',
+    require.resolve('./base.js')
+  ],
   env: {
     'browser': true
   },
   rules: {
-    'new-cap': [ 'error', { 'capIsNewExceptions': [ 'A' ] } ]
+    'new-cap': [ 'error', { 'capIsNewExceptions': [ 'A' ] } ],
+
+    // Custom rules
+    'ember/avoid-leaking-state-in-components': 'error',
+    'ember/local-modules': 'error',
```

Comment:
I think we should chuck this rule. Since this is a major release, any app that updates to it will have to do so explicitly, and that's a good opportunity to run the codemod to cut over to the new module imports (whose rule is enabled by default in the recommended set).